gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-debug/src/main/java/io/gravitee/gateway/debug/DebugConfiguration.java

```diff
@@ -144,46 +151,24 @@ public PolicyChainProviderLoader policyChainProviderLoader(
     @Bean
     @Qualifier("debugConfigurablePolicyChainProvider")
     public OnRequestPlatformPolicyChainProvider onRequestPlatformPolicyChainProvider(
```

Comment:
Yes because you worked on Jupiter and you know which bean exists or not for Jupiter. I was thinking about the other team members which haven't necessarily this knowledge and for whom it will be easier to have a bean name containing 'v3' to quickly understand that it is now only make sense in v3 and no longer in Jupiter. WDYT ?